ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -110,7 +111,16 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
               SystemClock.nanoTime(),
               createWebViewEvent(webView, url)));
     }
-
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        ReactContext reactContext = (ReactContext) view.getContext();
+        if(url.startsWith("tel:")) {
+            Intent intent = new Intent(Intent.ACTION_DIAL, Uri.parse(url)); 
+            reactContext.startActivity(intent);            
```

Comment:
Does it work without the flag Intent.FLAG_ACTIVITY_NEW_TASK? We should use the current activity to start the intent (which doesn't need the flag) if we can. It provides better UX since most of the time the user can press back to come to the previous screen.